core/base/inc/TROOT.h

```diff
@@ -317,6 +322,7 @@ friend TROOT *ROOT::Internal::GetROOT2();
    void              SaveContext();
    void              SetApplication(TApplication *app) { fApplication = app; }
    void              SetBatch(Bool_t batch = kTRUE) { fBatch = batch; }
+   void              SetWeb(TString web);
```

Comment:
Why a TString as argument to SetWeb? (and not even a ref...)

Reply:
Here also missing GetWeb() method.

Reply:
it should probably just be a std::string_view .... (and then a second overload being a TStringView to avoid ambiguity ..)